### PR TITLE
vscode: API version to 1.78.0

### PR DESCRIPTION
**Pull request by rschnekenbu**
#### What it does
This pull-request updates the default vscode api from 1.77.0 to 1.78.0. With #12530  main subtasks being completed, the API compatibility can be raised. 
The currently remaining issue #12560 is only a menu contribution entry, which will not block the API compatibility as such.

#### How to test
N/A

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)
Reminder for reviewers
- [x]  As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by JonasHelming** [comment]
@vince-fugnitto if possible, could we merge this before the release? If you see this after our business hours, just feel free to merge.

**Review by vince-fugnitto — APPROVED** [review]
LGTM :+1:

**Comment by rschnekenbu** [comment]
Thanks @vince-fugnitto !

**Pull request merged**
